src/main/java/com/google/api/codegen/transformer/ImportTypeTransformer.java

```diff
@@ -24,12 +24,13 @@
 import java.util.Set;
 import java.util.TreeSet;
 
-public class ImportTypeTransformer {
+public class ImportTypeTransformer implements ImportGenerator {
```

Comment:
I don't think we should have separate "transformer" and "generator" terms. I would suggest renaming this existing class `ImportTypeTransformer` to `StandardImportTypeTransformer` and renaming the interface `ImportGenerator` to `ImportTypeTransformer`.